extension/scripts/features/bazaar-sub-vendor-items/ttSubVendorPriceItems.css

```diff
@@ -0,0 +1,3 @@
+.tt-highlight-item {
```

Comment:
You should probably use a different classname. We already use this for another feature on the same page and reusing it means that disabling one feature will remove the highlights of the other feature.